gcp-observability/src/main/java/io/grpc/gcp/observability/ObservabilityConfigImpl.java

```diff
@@ -74,95 +87,128 @@ void parse(String config) throws IOException {
 
   private void parseConfig(Map<String, ?> config) {
     if (config != null) {
-      Boolean value = JsonUtil.getBoolean(config, "enable_cloud_logging");
-      if (value != null) {
-        enableCloudLogging = value;
-      }
-      value = JsonUtil.getBoolean(config, "enable_cloud_monitoring");
-      if (value != null) {
-        enableCloudMonitoring = value;
-      }
-      value = JsonUtil.getBoolean(config, "enable_cloud_trace");
-      if (value != null) {
-        enableCloudTracing = value;
-      }
-      destinationProjectId = JsonUtil.getString(config, "destination_project_id");
-      flushMessageCount = JsonUtil.getNumberAsLong(config, "flush_message_count");
-      List<?> rawList = JsonUtil.getList(config, "log_filters");
-      if (rawList != null) {
-        List<Map<String, ?>> jsonLogFilters = JsonUtil.checkObjectList(rawList);
-        ImmutableList.Builder<LogFilter> logFiltersBuilder = new ImmutableList.Builder<>();
-        for (Map<String, ?> jsonLogFilter : jsonLogFilters) {
-          logFiltersBuilder.add(parseJsonLogFilter(jsonLogFilter));
-        }
-        this.logFilters = logFiltersBuilder.build();
+      projectId = JsonUtil.getString(config, "project_id");
+      parseLoggingObject(config, CONFIG_LOGGING_VAR_NAME);
+      parseMonitoringObject(config, CONFIG_MONITORING_VAR_NAME);
+      parseTracingObject(config, CONFIG_TRACING_VAR_NAME);
+      parseCustomTags(config, CONFIG_LABELS_VAR_NAME);
+    }
+  }
+
+  private void parseLoggingObject(Map<String, ?> config,  String jsonObjectName) {
+    Map<String, ?> rawCloudLogging = JsonUtil.getObject(config, jsonObjectName);
```

Comment:
nit: It seems strange to me to pass in the object name to these methods. I'd have either expected the caller to call getObject() and pass in the map.

I could have understood passing in the name for logging in order to add it to any error messages, but that wasn't done and that wouldn't apply to the other methods.

Reply:
It was added to be used in logging, but since observability does not log in other places not sure if its okay to just log in this file alone. Is there any recommended logging convention we follow, like log only for errors or info level as well?

Reply:
What kind of info messages? For error checking you have things like `checkArgument` which seems to be okay for now - just that it will throw an exception.

Reply:
> ... since observability does not log in other places ...

In InternalLoggingChannelInterceptor I found things like

```
logger.log(Level.SEVERE, "Unable to log request header", e);
```

So we do log in o11y code in other places